### ISSUE TITLE
EI_EXPOSE_REP2: Inconsistent reporting

### ISSUE THREAD

**Issue by sco0ter** [spotbugs/spotbugs#603]
EI_EXPOSE_REP2 is correctly reported when using this:

```java
public Foo(byte[] bytes) {
    this.bytes = bytes;
}
```
but it's not reported when using this:
```java
public Foo(byte[] bytes) {
    this.bytes = Objects.requireNonNull(bytes);
}
```

Expected behavior: EI_EXPOSE_REP2 is reported in both cases, because it's essentially the same.

Spotbugs Version: 3.1.3

**Comment by KengoTODA** [comment]
> Spotbugs Version: 3.1.3

Do you mean 3.1.2? For now 3.1.3 is not released yet.

**Comment by sco0ter** [comment]
I mean the Maven plugin:
https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-maven-plugin/3.1.3

I assumed the versioning of the Maven plugin is consistent with the versioning of SpotBugs.

**Comment by KengoTODA** [comment]
This is because `OpcodeStack.Item#isInitialParameter()` returns false for `Objects.requireNonNull(bytes)`. See `FindReturnRef#isPotentialCapture()` for detail.

I think we can fix not `FindReturnRef` but `OpcodeStack#sawOpcode()`.
